controllers/secretdefinition_controller_test.go

```diff
@@ -12,8 +12,10 @@ import (
 
```

Comment:
I'll will take more care of the test suite in the next steps. https://github.com/tuenti/secrets-manager/issues/79